### PR TITLE
Chore: bump stream-chat

### PR DESCRIPTION
**Pull request by mahboubii**
# Submit a pull request

## CLA

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [ ] The code changes follow best practices
- [ ] Code changes are tested (add some information if not applicable)

### CODE REVIEW

**Review by vishalnarkhede — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -164,6 +164,7 @@
     "react-native-web": "0.14.1",
     "react-styleguidist": "11.1.0",
     "react-test-renderer": "16.13.1",
+    "stream-chat": "^2.7.0",
```
Comment (virdesai) [inline package.json:167]:
```suggestion
    "stream-chat": "2.7.0",
```

File: package.json [hunk 2]
```diff
@@ -102,14 +102,14 @@
...
-    "stream-chat": "2.6.0",
     "styled-components": "5.2.0",
     "uuid": "8.3.1"
   },
   "peerDependencies": {
     "react": "^15.0.0 || ^16.0.0",
     "react-dom": "^15.0.0 || ^16.0.0",
-    "react-native": "^0.60.0"
+    "react-native": "^0.60.0",
+    "stream-chat": "2.7.x"
```
Comment (virdesai) [inline package.json:112]:
```suggestion
    "stream-chat": "^2.7.0"
```

File: yarn.lock [hunk 3]
```diff
@@ -11568,10 +11568,10 @@ stream-buffers@~2.2.0:
   resolved "https://registry.yarnpkg.com/stream-buffers/-/stream-buffers-2.2.0.tgz#91d5f5130d1cef96dcfa7f726945188741d09ee4"
   integrity sha1-kdX1Ew0c75bc+n9yaUUYh0HQnuQ=
 
-stream-chat@2.6.0:
-  version "2.6.0"
-  resolved "https://registry.yarnpkg.com/stream-chat/-/stream-chat-2.6.0.tgz#2fae6e4c5a56a8f86dee0676654fe2511d751aab"
-  integrity sha512-hI1aIMCkcGektSOt7Gt2CrrKmHKn31ovbJ+IWz1rsgdZHiWLCG2QG7JgGUKeoJVD/Ix501CjnEugZHlFdbZLwg==
+stream-chat@^2.7.0:
```
Comment (virdesai) [inline yarn.lock:11571]:
```suggestion
stream-chat@2.7.0:
```

**Comment by virdesai** [comment]
why are you moving it to a peerDependency?

File: package.json (hunk 2 above)
Comment (mahboubii) [reply package.json:112]: > why are you moving it to a peerDependency?

isn't the usage the same as react components? we don't specifically import it and the user supposed to install the `stream-chat` and pass it down to the components

Comment (mahboubii) [reply package.json:112]: IMO better to lock into patch upgrade only, minor updates can be breaking too sometimes if we miss something

**Comment by mahboubii** [comment]
> why are you moving it to a peerDependency?

isn't the usage the same as react components? we don't specifically import it and the user supposed to install the stream-chat and pass it down to the components

**Review by virdesai — APPROVED** [review]

**Pull request merged**